_rules/iframe-non-empty-accessible-name-cae760.md

```diff
@@ -142,6 +146,14 @@ This `iframe` element has a negative `tabindex` [attribute value][].
 <iframe tabindex="-1" src="/test-assets/SC4-1-2-frame-doc.html"> </iframe>
```

Comment:
Hmm, this one needs a height so that it isn't scrollable. 

https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/cae760/803b52f32a5d2193683f152aa5e00174d5f8416a.html

Reply:
Added